modules/fluid_properties/test/tests/brine/tests

```diff
@@ -5,5 +5,19 @@
     csvdiff = 'brine_out.csv'
     allow_test_objects = true
     threading = '!pthreads'
+    requirement = 'BrineFluidProperties shall correctly calculate brine properties'
+    design = 'BrineFluidProperties.md'
+    issues = '#6972'
+  [../]
+  [./brine_tabulated]
+    type = CSVDiff
+    input = 'brine_tabulated.i'
+    csvdiff = 'brine_out.csv'
+    allow_test_objects = true
+    threading = '!pthreads'
+    prereq = brine
+    requirement = 'BrineFluidProperties shall correctly calculate brine properties using a supplied water properties UserObject'
```

Comment:
"The system shall computed brine properties using tabulated values of water properties." - That's my guess what you are trying to test?